cmd/wallet-web/src/pages/chapi/Login.vue

```diff
@@ -111,8 +121,36 @@ SPDX-License-Identifier: Apache-2.0
             loginDevice: async function () {
                 await this.deviceLogin.login();
             },
+            getCookie (name) {  /// Retrieve cookie function
+               var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
```

Comment:
Why is the regexp needed?

Reply:
this is no longer required, missed in the cleanup. removed now.